src/main/CommandHandler.cpp

```diff
@@ -1051,51 +1048,6 @@ CommandHandler::checkBooted() const
     }
 }
 
-void
-CommandHandler::surveyTopology(std::string const& params, std::string& retStr)
-{
-    ZoneScoped;
-
-    CLOG_WARNING(
-        Overlay,
-        "`surveytopology` is deprecated and will be removed in a future "
-        "release.  Please use the new time sliced survey interface.");
-
-    checkBooted();
-
-    std::map<std::string, std::string> map;
-    http::server::server::parseParams(params, map);
-
-    auto duration =
-        std::chrono::seconds(parseRequiredParam<uint32>(map, "duration"));
-    auto idString = parseRequiredParam<std::string>(map, "node");
-    NodeID id = KeyUtils::fromStrKey<NodeID>(idString);
-
-    auto& surveyManager = mApp.getOverlayManager().getSurveyManager();
-
-    bool success = surveyManager.startSurveyReporting(
-        SurveyMessageCommandType::SURVEY_TOPOLOGY, duration);
-
-    surveyManager.addNodeToRunningSurveyBacklog(
-        SurveyMessageCommandType::SURVEY_TOPOLOGY, duration, id, std::nullopt,
-        std::nullopt);
-    retStr = "Adding node.";
-
-    retStr += success ? "Survey started " : "Survey already running!";
-}
-
-void
-CommandHandler::stopSurvey(std::string const&, std::string& retStr)
```

Comment:
We use `stopSurvey` in the survey script for retry purposes, so we should keep this endpoint (and probably remove the deprecation warning. You can rename the command if you'd like, but I think we should keep the ability to stop the reporting phase)
https://github.com/stellar/stellar-core/blob/69cb5cb2e8cb5a6d8c09bf4df7c3ef8505331357/scripts/OverlaySurvey.py#L408

Reply:
Ah, good catch! I removed it because it was deprecated, not realizing that we were still using it. Turns out I added that functionality to the script *after* deprecating the endpoint. I should have un-deprecated it then. Whoops. I added the endpoint back and removed the deprecation warning. I agree it's important to keep.